src/Advertising.js

```diff
@@ -187,9 +192,27 @@ export default class Advertising {
         });
     }
 
+    [defineOutOfPageSlot]() {
+        if (this.config.outOfPage) {
```

Comment:
So you are adding a new property `outOfPage` to the root of the config, with just an ID that references the out of page slots.

I would not do it this way, since it's redundant and prevents users from defining more than one out of page slot.

You should iterate the out of page slot configs instead. 

Let's have a quick discussion in person about this.